apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html

```diff
@@ -906,9 +906,8 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
 
                             <div class="mrR1 form-group pull-left cp">
                                 <label class="pull-left cp fs16">
-                                    <input class="pull-left" type="checkbox" name="generateInvoice"
-                                        [(ngModel)]="vm.selectedLedger.generateInvoice"
-                                        [disabled]="vm.isInvoiceGeneratedAlready">
+                                    <input class="pull-left" type="checkbox" name="generateInvoice" [(ngModel)]="vm.selectedLedger.generateInvoice"
+                                           [disabled]="vm.isInvoiceGeneratedAlready || (isAdvanceReceipt && vm.selectedLedger.generateInvoice)">
```

Comment:
Consider using `disabled` as:
```suggestion
                                           [disabled]="(vm.isInvoiceGeneratedAlready || (isAdvanceReceipt && vm.selectedLedger.generateInvoice))' ? 'disabled' : null">
```
It will ensure that disabled attribute is removed when condition is false

Reply:
I haven't make change in this line